unlock-app/src/components/interface/Error.js

```diff
@@ -6,52 +6,45 @@ import { resetError } from '../../actions/error'
 import Buttons from './buttons/layout'
 import ErrorMessage from '../helpers/ErrorMessage'
 
-export const Error = ({ error, close }) => {
-  if (!error) {
+export const Errors = ({ errors, close }) => {
+  const content = errors.map(error => (
+    <Wrapper key={error}>
+      <Error>{ErrorMessage(error)}</Error>
+      <SecondColumn cols={errors.length ? errors.length : 1}>
+        <Buttons.Close as="button" onClick={close} size="16px">
```

Comment:
So we're back to a single `close` function? I thought each cross should close its corresponding error?

Reply:
this is a casualty of a crappy rebase :(